app/src/main/res/layout/fragment_quick_settings_dialog_sheet.xml

```diff
@@ -52,7 +54,7 @@
             style="@style/QuickSettingsText.Icon"
             android:layout_width="match_parent"
             android:layout_height="@dimen/quicksettings_item_height"
-            android:drawableStart="@drawable/ic_tracking_protection"
+            tools:drawableStart="@drawable/ic_tracking_protection"
```

Comment:
Based on this change, I'm assuming we don't want to show an icon by default then?

Reply:
We set this icon in code. It looked off when using `android:drawableStart` and removing `putCompoundDrawables` so I went with the programmatic way.